tools/wpt/run.py

```diff
@@ -327,6 +327,29 @@ def setup_kwargs(self, kwargs):
                 raise WptrunError("Unable to locate or install operadriver binary")
 
 
+class EdgeChromium(BrowserSetup):
+    name = "MicrosoftEdge"
```

Comment:
```suggestion
    name = "EdgeChromium"
```

…rather than having multiple things claiming to be MicrosoftEdge.

Reply:
Due to some legal issues, we have to keep the same name for the browser name. Also, MSEdgeDriver is checking the browser name against "MicrosoftEdge" in the capability, so changing that will make the WebDriver not working correctly.

Reply:
Okay.